src/Pages/AdminPage/AdminPage.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import styled from 'styled-components'
+import {Link} from 'react-router-dom'
+import Logo from "../../assets/images/LogoMegaK.png";
+
+
+const Wrapper = styled.div`
+  width: 100vw;
+  min-height: 100vh;
+  background-color: #222224;
+  display: flex;
+  justify-content: center;
+  gap:15rem;
+  flex-direction: row;
+  overflow: hidden;
+
+  @media screen and  (max-width: 768px) {
+    
+gap:7rem;
+}
+`
+//logo MegaK right-top
+export const LogoMegaKRT = styled.div`
+  display: flex;
+  position: absolute;
+  top: 5rem;
+  right: 5rem;
+  background-image: url(${Logo});
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-bottom: 30px;
+  min-width: 120px;
+  min-height: 75px;
+  width: 10vw;
+  height: 7vw;
+`;
+
```

Comment:
W tym wypadku starałbym sie użyc komponentów ktore już mamy, jeżeli ich za specjalnie nie modyfikujesz i nie musisz pchać bardzo dużej ilosci propsów :) 
Ewentualnie jeżeli potrzebujesz cos lekko zmienic zawsze można zrobic props który zmieni Ci cos a jezeli go nie bedzie to zostawi wartosc domyslna cos takiego 

margin-bottom : ${({margin)=> margin || "30px"};